roles/mysqld_exporter/defaults/main.yml

```
@@ -0,0 +1,35 @@
+---
+mysqld_exporter_version: 0.14.0
+mysqld_exporter_binary_local_dir: ""
+mysqld_exporter_binary_url: "https://github.com/prometheus/mysqld_exporter/releases/download/v{{ mysqld_exporter_version }}/\
+                           mysqld_exporter-{{ mysqld_exporter_version }}.linux-{{ go_arch }}.tar.gz"
+mysqld_exporter_checksums_url: "https://github.com/prometheus/mysqld_exporter/releases/download/v{{ mysqld_exporter_version }}/sha256sums.txt"
+
+mysqld_exporter_web_listen_address: "0.0.0.0:9104"
+mysqld_exporter_web_telemetry_path: "/metrics"
+
+mysqld_exporter_username: exporter
+mysqld_exporter_password: secret
+
+mysqld_exporter_config_file: "mysqld_exporter.cnf"
+# mysqld_exporter_host: localhost
+# mysqld_exporter_port: 3306
+mysqld_exporter_socket: "/run/mysqld/mysqld.sock"
+
+mysqld_exporter_tls_server_config: {}
+
+mysqld_exporter_http_server_config: {}
+
+mysqld_exporter_basic_auth_users: {}
+
+mysqld_exporter_enabled_collectors: []
+#  - engine_innodb_status
+#  - perf_schema.eventsstatements
+
+mysqld_exporter_disabled_collectors: []
+
+# Internal variables.
+_mysqld_exporter_binary_install_dir: "/usr/local/bin"
+_mysqld_exporter_config_dir: "/etc/mysqld_exporter"
+_mysqld_exporter_system_group: "mysqld-exp"
+_mysqld_exporter_system_user: "{{ _mysqld_exporter_system_group }}"
```

```suggestion
mysqld_exporter_binary_install_dir: "/usr/local/bin"
mysqld_exporter_config_dir: "/etc/mysqld_exporter"
mysqld_exporter_system_group: "mysqld-exp"
mysqld_exporter_system_user: "{{ mysqld_exporter_system_group }}"
```

Same reasoning as #65 
Also needs to be added to meta/argument_specs.yml

---

I'm not sure that we want to do #65. We had reasons to make those vars `_` prefixed.